pages/pipelines/environment_variables.md.erb

```diff
@@ -303,6 +303,18 @@ The organization name on Buildkite as used in URLs. The value cannot be modified
 
 Example: `"acme-inc"`
 
+<h3 class="h3-caps">BUILDKITE_PARALLEL_JOB</h3>
+
+The number of parallel jobs created from a parallel build step. Starting from 0. For a build step with `parallelism: 5`, the value would be 0, 1, 2, 3, and 4 respectively.
```

Comment:
So this is the "index" for a specific job created from a parallel step? 

We might need something like

The index of each parallel job created from a parallel build step. Starts from 0 and has a maximum of (total parallelism -1 ).  For a build step with `parallelism: 5`, the value for each job would be 0, 1, 2, 3, and 4 respectively.